lib/plugins/servercookie.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Treasure Server Side Cookie
+ */
+
+var jsonp = require('jsonp')
+var noop = require('../utils/misc').noop
+var invariant = require('../utils/misc').invariant
+var cookie = require('../vendor/js-cookies')
+
+var cookieName = '_td_ssc_id'
+
+function configure () {
```

Comment:
@MikeMcElroy should it automatically fetch serverSideCookie here? I don't think it should

Reply:
I guess since the model we're going for is like `td_global_id`, we shouldn't, but we should confirm with Sada that that is expected behavior.  It does force the customer to change their startup routine from what it currently is -- as in, they'd need to explicitly call `fetchServerCookie` instead of it happening automagically.